RecoParticleFlow/PFProducer/plugins/importers/GeneralTracksImporter.cc

```diff
@@ -21,7 +21,9 @@ class GeneralTracksImporter : public BlockElementImporterBase {
         cleanBadConvBrems_(
             conf.existsAs<bool>("cleanBadConvertedBrems") ? conf.getParameter<bool>("cleanBadConvertedBrems") : false),
         debug_(conf.getUntrackedParameter<bool>("debug", false)) {
-    pfmu_ = std::unique_ptr<PFMuonAlgo>(new PFMuonAlgo(conf));
+    bool postMuonCleaning =
+        conf.existsAs<bool>("postMuonCleaning") ? conf.getParameter<bool>("postMuonCleaning") : false;
+    pfmu_ = std::unique_ptr<PFMuonAlgo>(new PFMuonAlgo(conf, postMuonCleaning));
```

Comment:
Why is this existsAs needed here (and in some other locations) if the parameter is part of fillDescriptions ayway?

Reply:
This comes from PFBlockProducer.cc (not PFProducer.cc) for which we haven't introduced fillDescriptions yet...